backends/pkg/reconciler/eventmesh/controller.go

```diff
@@ -59,7 +61,17 @@ func startWebServer(ctx context.Context, listers Listers) {
 	r := mux.NewRouter()
 	r.Use(commonMiddleware)
 
-	r.HandleFunc("/", HttpHandler(ctx, listers)).Methods("GET")
+	noTokenConfig, err := rest.InClusterConfig()
+	noTokenConfig.BearerToken = ""
+	noTokenConfig.Username = ""
+	noTokenConfig.Password = ""
+	noTokenConfig.BearerTokenFile = ""
+
+	if err != nil {
+		log.Fatalf("Error getting in-cluster config: %v", err)
+	}
```

Comment:
`err` should be checked as soon as we get it to avoid panics for `nil` pointer

Reply:
Also the suggestion from @aliok was to copy the config
```
	config, err := rest.InClusterConfig()
	if err != nil {
		log.Fatalf("Error getting in-cluster config: %v", err)
	}
	noTokenConfig := rest.CopyConfig(config)
	noTokenConfig.BearerToken = ""
	noTokenConfig.Username = ""
	noTokenConfig.Password = ""
	noTokenConfig.BearerTokenFile = ""
```

Reply:
`InClusterConfig` function actually always creates a new copy, so, it is ok

Reply:
This is done now